src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement/Program.cs

```diff
@@ -1,18 +1,29 @@
 using System.Diagnostics.CodeAnalysis;
 using Altinn.AccessManagement;
 using Altinn.AccessMgmt.Persistence.Extensions;
-using Microsoft.FeatureManagement;
+using Microsoft.Extensions.Options;
 using Microsoft.IdentityModel.Logging;
 
 AppDomain domain = AppDomain.CurrentDomain;
 domain.SetData("REGEX_DEFAULT_MATCH_TIMEOUT", TimeSpan.FromSeconds(2));
 
 WebApplication app = AccessManagementHost.Create(args);
 
-if (await app.Services.GetRequiredService<FeatureManager>().IsEnabledAsync(AccessManagementFeatureFlags.MigrationDb))
+var appsettings = app.Services.GetRequiredService<IOptions<AccessManagementAppsettings>>().Value;
+if (appsettings.Database.MigrateDb || appsettings.Database.MigrateDbAndTerminate)
 {
-    bool generateBasicData = await app.Services.GetRequiredService<FeatureManager>().IsEnabledAsync(AccessManagementFeatureFlags.MigrationDbWithBasicData);
-    await app.UseAccessMgmtDb(generateBasicData);
+    using var scope = app.Services.CreateScope();
+
+    // Do Something like this when using EF
+    // scope.ServiceProvider.GetRequiredService<AppDbContext>().MigrateAsync();
+
+    // Current framework
+    await app.UseAccessMgmtDb(true);
```

Comment:
Default `true` for AccessManagementFeatureFlags.MigrationDbWithBasicData (som brukes bare for integrasjonstester på test container m/testdata) blir vel feil. Flagget må vel evt. også flyttes inn i DatabaseSettings?

Reply:
denne skal egentlig fjernes. PR var ikke helt klar, burde satt den til draft :D